### PR TITLE
Bug/add update locations

### PR DESCRIPTION
**Pull request by RishiRamraj**
Add a script that updates the names of locations. Note that proper csv parsing isn't well supported by these scripts.

### CODE REVIEW

File: examples/bash/tools/update_locations [hunk 1]
```diff
@@ -0,0 +1,93 @@
...
+    del(.memberCount) |
+    del(.origin)
+  "
+  location=$(echo $locations | jq "$filter")
+
+  # Check for a difference in name.
+  current_name=$(echo $location | jq -r ".name")
+  if [ "$name" = "$current_name" ]; then
+    echo "$id | Names equal: \"$current_name\""
+    continue
```
Comment (mrname) [inline examples/bash/tools/update_locations:58]: So, if the name does not change, we don't change anything else?

File: examples/bash/tools/update_locations [hunk 2]
```diff
@@ -0,0 +1,93 @@
+#!/bin/bash
+set -e
+
+# Validate arguments.
+if [ $# -ne 2 ]
+then
+  echo "./csv2shifts <token> <csv>"
+  echo "Update location names."
+  echo "csv format: id,name,address,timezone,ip_address_list,phone,deleted,deleted_by"
```
Comment (mrname) [inline examples/bash/tools/update_locations]: It seems like the script only updates the name and members... if this is correct, why do we include all the other stuff in the CSV?

File: examples/bash/tools/update_locations [hunk 3]
```diff
@@ -0,0 +1,93 @@
+#!/bin/bash
+set -e
+
+# Validate arguments.
+if [ $# -ne 2 ]
+then
+  echo "./csv2shifts <token> <csv>"
```
Comment (mrname) [inline examples/bash/tools/update_locations]: `csv` here is path to a file?

File: examples/bash/tools/update_locations (hunk 1 above)
Comment (RishiRamraj) [reply examples/bash/tools/update_locations:58]: Yup. Adding a comment.

File: examples/bash/tools/update_locations (hunk 2 above)
Comment (RishiRamraj) [reply examples/bash/tools/update_locations]: I've removed them.

File: examples/bash/tools/update_locations (hunk 3 above)
Comment (RishiRamraj) [reply examples/bash/tools/update_locations]: Yup, that's correct. Updating the variable name.

**Pull request merged**